models/apartment.rb

```diff
@@ -0,0 +1,13 @@
+class Apartment
+  attr_reader (:id, :address, :rent, :square_feet)
+  def initialize(id, address, rent, square_feet)
+    @id = id
+    @address = address
+    @rent = rent
+    @square_feet = square_feet
+  end
+  def Apartment.new
```

Comment:
We generally don't want to redefine the `Apartment.new` method, that's provided to us by Ruby.